junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/Constants.java

```diff
@@ -463,6 +463,17 @@
 	@API(status = EXPERIMENTAL, since = "5.7")
 	public static final String DEFAULT_TEST_METHOD_ORDER_PROPERTY_NAME = JupiterConfiguration.DEFAULT_TEST_METHOD_ORDER_PROPERTY_NAME;
 
+	/**
+	 * Property name used to set the class orderer class name: {@value}
+	 *
+	 * <h3>Supported Values</h3>
+	 *
+	 * <p>Supported values include fully qualified class names for types that
+	 * implement {@link org.junit.jupiter.api.ClassOrderer}.
+	 */
+	@API(status = EXPERIMENTAL, since = "5.8")
+	public static final String CUSTOM_CLASS_ORDERER_PROPERTY_NAME = JupiterConfiguration.CUSTOM_CLASS_ORDERER_PROPERTY_NAME;
```

Comment:
I think this property should be called `DEFAULT_TEST_CLASS_ORDER_PROPERTY_NAME` (same for `JupiterConfguration`) for consistency with `DEFAULT_TEST_METHOD_ORDER_PROPERTY_NAME` and its value.

Reply:
done